transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -194,8 +194,7 @@ private void decrementPendingOutboundBytes(long size, boolean invokeLater, boole
         }
 
         long newWriteBufferSize = TOTAL_PENDING_SIZE_UPDATER.addAndGet(this, -size);
-        if (notifyWritability && (newWriteBufferSize == 0
-            || newWriteBufferSize <= channel.config().getWriteBufferLowWaterMark())) {
+        if (notifyWritability && newWriteBufferSize < channel.config().getWriteBufferLowWaterMark()) {
```

Comment:
@buchgr you removed the `newWriteBufferSize == 0` as someone could configure 0 ?